Vue: Status Bar component missing

### Initial checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Forum](https://community.transloadit.com/)
- [x] I searched [issues](https://github.com/transloadit/uppy/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Link to runnable example

_No response_

### Steps to reproduce

The package `@uppy/vue@3.0.0` does not contain Status Bar code.

While attempting to update to the new Uppy and adjust the import statements, my IDE was able to autocomplete to the StatusBar location, but with the error `Cannot find module '@uppy/vue/status-bar' or its corresponding type declarations.`.

Looking at `package.json`, I am able to see the export:

```json
  "exports": {
    ".": "./lib/index.js",
    "./css/style.css": "./dist/styles.css",
    "./dashboard": "./lib/dashboard.js",
    "./dashboard-modal": "./lib/dashboard-modal.js",
    "./status-bar": "./lib/status-bar.js",
    "./package.json": "./package.json"
  },
```

However, the file `./lib/status-bar.js` is no where to be found in the package as of version 3.0.0.

### Expected behavior

The Status Bar and related code do be included in the package `@uppy/vue`.

### Actual behavior

The Status Bar and related code are not included in the package `@uppy/vue`.

--- a/.changeset/odd-goats-bow.md
+++ b/.changeset/odd-goats-bow.md
@@ -1,0 +1,6 @@
+---
+"@uppy/react": patch
+"@uppy/vue": patch
+---
+
+Remove status-bar from export map

--- a/packages/@uppy/react/package.json
+++ b/packages/@uppy/react/package.json
@@ -59,7 +59,6 @@
     "./css/style.css": "./dist/styles.css",
     "./dashboard": "./lib/Dashboard.js",
     "./dashboard-modal": "./lib/DashboardModal.js",
-    "./status-bar": "./lib/StatusBar.js",
     "./package.json": "./package.json"
   },
   "peerDependencies": {

--- a/packages/@uppy/vue/package.json
+++ b/packages/@uppy/vue/package.json
@@ -33,7 +33,6 @@
     "./css/style.css": "./dist/styles.css",
     "./dashboard": "./lib/dashboard.js",
     "./dashboard-modal": "./lib/dashboard-modal.js",
-    "./status-bar": "./lib/status-bar.js",
     "./package.json": "./package.json"
   },
   "peerDependencies": {

